helm/adelphi/templates/schema-job.yaml

```diff
@@ -45,4 +45,4 @@ spec:
         secretName: {{ .Values.target.clusterName }}-superuser
         items:
         - key: password
-          path: password
\ No newline at end of file
+          path: password
```

Comment:
A side effect of an earlier impl which tried to perform schema extraction in the schema-job template